lab2.js

```diff
@@ -99,13 +138,22 @@ function sayHello (sb) {
     // to do the translating
 
     //TODO: put this on the SentientBeing prototype
-  }
 
 // TODO: create three subclasses of SentientBeing, one for each
```

Comment:
the asserts are failing because they're not constructor functions (I got a reference error on the new Human() when I ran it). Right now, Human is just an instance of Sentient Being. You need to create a constructor function for humans and add an instance of a sentient being as its prototype ( you could use your instance of Human that you made, above).